packages/flame/lib/src/components/core/component.dart

```diff
@@ -580,6 +580,16 @@ class Component {
     components.forEach(remove);
   }
 
+  /// Removes all the children the [test] function returns true for and calls
+  /// [onRemove] for all of them and their children.
```

Comment:
```suggestion
  /// Removes all the children for which the [test] function returns true.
```

Reply:
This was just copy paste from the other method 😅